packages/array/index.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Create a numeric range
+ *
+ * @param {Number} from
+ * @param {Number} to
+ * @return {Array}
+ *
+ * @example
+ * Array.range(-2, 2) // => [-2, -1, 0, 1, 2]
+ * Array.range(2, -2) // => [2, 1, 0, -1, -2]
+ */
+export declare function range(from: number, to: number): any[];
```

Comment:
The return is always numbers, isn't it?

```suggestion
export declare function range(from: number, to: number): number[];
```